site/content/docs/manifest/environment.en.md

```diff
@@ -145,6 +145,59 @@ An IPv4 CIDR block assigned to the subnet. This field is mutually exclusive with
 The Availability Zone name assigned to the subnet. The `az` field is optional, by default Availability Zones are assigned in alphabetical order.
 This field is mutually exclusive with `id`.
 
+<span class="parent-field">network.vpc.</span><a id="network-vpc-security-group" href="#network-vpc-security-group" class="field">`security_group`</a> <span class="type">Map</span>  
+Rules of the environment's security group.
+
+<span class="parent-field">network.vpc.security_group</span><a id="network-vpc-security-group-ingress" href="#network-vpc-security-group-ingress" class="field">`ingress`</a> <span class="type">Map</span>  
+Rules controlling inbound traffic.
+```yaml
+network:
+  vpc:
+    security_group:
+      ingress:
+        - ip_protocol: tcp
+          ports: 0-65535  
+          cidr: 0.0.0.0/0
+```
+
+<span class="parent-field">network.vpc.security_group</span><a id="network-vpc-security-group-egress" href="#network-vpc-security-group-egress" class="field">`egress`</a> <span class="type">Map</span>  
+Rules controlling outbound traffic.
+```yaml
+network:
+  vpc:
+    security_group:
+      egress:
+        - ip_protocol: tcp
+          ports: 80
+          cidr: 0.0.0.0/0
+```
+<span class="parent-field">network.vpc.security_group.</span><a id="network-vpc-security-group-ingress" href="#network-vpc-security-group" class="field">`ingress`</a> <span class="type">Array of Security Group Rules</span>    
+A list of inbound security group rules.
+
+<span class="parent-field">network.vpc.security-group.</span><a id="network-vpc-security-group-egress" href="#network-vpc-security-group" class="field">`egress`</a> <span class="type">Array of Security Group Rules</span>    
+A list of outbound security group rules.
+
+
+<span class="parent-field">network.vpc.security_group.<type\>.</span><a id="network-vpc-security-group-ip-protocol" href="#network-vpc-security-group-ip-protocol" class="field">`ip_protocol`</a> <span class="type">String</span>    
+The IP protocol name.
+
+<span class="parent-field">network.vpc.security_group.<type\>.</span><a id="network-vpc-security-group-ports" href="#network-vpc-security-group-ports" class="field">`ports`</a> <span class="type">String or Integer</span>     
+You can specify FromPort and ToPort for an environment security group rule.
```

Comment:
`FromPort` and `ToPort` are the CFN field name and my concern is that they seem to appear here out of context. What you do think of
```suggestion
The port range for the security group rule. 
```

Reply:
maybe just say `The port range or number for the security group rule.`? and change the section  `Alternatively, you can specify it as an Integer if both FromPort and ToPort is same` to just `or`?